clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -719,6 +732,11 @@ public boolean matches(AbstractRequest body) {
 
     @Test
     public void testUpdateLastHeartbeatPollWhenCoordinatorUnknown() throws Exception {
+        GroupRebalanceConfig rebalanceConfig = buildRebalanceConfig(Optional.empty());
```

Comment:
Also these declarations are masking member fields. We should avoid that because it might lead to bugs that are hard to detect. If we have member fields for any of the variables we need to customize in a test case, we should use those and avoid masking with local variables.

Reply:
+1